### DISCUSSION TITLE
Max Container Reach -- when all workers are under 200 containers and "Running"

### DISCUSSION THREAD

**Discussion by AlexandreODelisle** [concourse/concourse#6151] [General]
Concourse v6.3 
Deployed via Bosh on Xenial 621.84

Hi, 

We are currently having issue that can't sems to be explainable.. We are habing Max Containers Reach, behavior that we haven't had in a long time since upgrading the Concourse version ( great! ) 

It seems that even if we are having "running"nand none "stalled" workers from the Bosh point of view, we are still having Max Containers Reach.. 

The monitoring in Grafana isn't showing any Worker reaching higher than 200 Containers, nor does the _`fly -t main workers`_ ..

Looking at the web logs, I can see the 

> {"timestamp":"2020-10-13T23:10:43.181157141Z","level":"info","source":"atc","message":"atc.tracker.notify.run.finish.errored","data":{"build":2837516,"error":"max containers reached","job": "my_job","pipeline":"my_pipeline","session":"25.2401.8.10"}}
> 

How is it possible to link a Build # or session Id to an actual worker.. 

_`fly -t main builds`_  does not show the Worker information either .. 

Thanks for you help,

**Comment by skreddy6673** [comment]
concourse v6.5.1 will show worker info in the UI.

fly -t main containers will show the containers and workers.

One solution might be increasing the workers count, I know this might not be the best/only solutions but should help.
